java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java

```diff
@@ -0,0 +1,61 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.GeoAddOptions;
+import glide.api.models.commands.GeospatialData;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Geospatial Indices Commands" group for standalone and
+ * cluster clients.
+ *
+ * @see <a href="https://redis.io/commands/?group=geo">Geospatial Indices Commands</a>
+ */
+public interface GeospatialIndicesBaseCommands {
+
+    /**
+     * Adds geospatial members with their positions to the specified sorted set stored at <code>key
+     * </code>.<br>
+     * If a member is already a part of the sorted set, its position is updated.
+     *
+     * @see <a href="https://redis.io/commands/geoadd/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param membersToGeoSpatialData A mapping of member names to their corresponding positions. See
+     *     {@link GeospatialData}. The command will report an error when the user attempts to index
```

Comment:
drop sentence about the error

Reply:
ack
we don't write about errors in other commands
I'm ok to keep it until next review round